RELEASING.md

```diff
@@ -259,36 +276,33 @@ as described in the section above.
    make release-publish
    ```
 
-1. Merge the PR. This will cause candidate.docs.projectcalico.org to be updated (after a few minutes). Validate that everything looks correct before proceeding to the next step.
+1. Merge the PR. This will cause docs.projectcalico.org to be updated (after a few minutes). Validate that everything looks correct.
 
-1. Checkout the master branch
+## Adding the previous release to docs.projectcalico.org/archive
```

Comment:
Do we need to do this before we promote the candidate release? I'm worried there will be a blip where the 'previous release' is not built or published because it's no longer the production branch and it hasn't been added as a 'previous release' branch.

I'm wondering what netlify will do if a branch is both listed as the production branch _and_ listed as a branch build. A quick check of the UI and it doesn't seem like they let you specify it in both.

How hard would it be to prevent this blip? If it's too hard, I think we just note that this is one small but unfortunate side effect of this new process

Reply:
yeah.. it doesn't let us specify a branch as both. I don't see an easy way out unfortunately.. We can  make the production branch commit, a little earlier while doing the release to shorten the time.. but it'll still be a branch build away in terms of time.